### PR TITLE
Revert "Merge pull request #20 from Tobias-Fischer/patch-1"

### LINKED ISSUE #21: New build of yaml has wrong library name
- [ ] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue:

The new build of `yaml-feedstock` using cmake creates a library called `libyaml.2.dylib` but the name should be `libyaml-0.2.dylib` (or so for linux).  This has come up before in #13 that cmake doesn't properly create the versioned libraries. 

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

```
</details>

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
This reverts commit 76ec8ee3ee15d4258206c369d903af7963922681, reversing
changes made to f2fda6529bd4b101cd2ba893e8eb83b569b0c715.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Fix https://github.com/conda-forge/yaml-feedstock/issues/21

See https://github.com/conda-forge/yaml-feedstock/pull/22 - it seems like the better way (at least for now) is to go back to the autotools build.

I am unsure about:
- whether to rerender and bump the build number (if so, bump by 2 as the broken cmake build was a bump by 1?)
- whether to merge as is, and if so whether to add `[ci skip] [skip ci] [cf admin skip] ***NO_CI***`?

@conda-forge/yaml - please feel free to edit the PR as needed or let me know which option you prefer.

Sorry again for messing up the cmake build!

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,10 +37,6 @@ test:
     - if not exist "%LIBRARY_LIB%\\yaml.lib" exit 1   # [win]
     - if not exist "%LIBRARY_BIN%\\yaml.dll" exit 1   # [win]
 
-    # Check cmake and pkg-config scripts
-    - test -f "${PREFIX}/cmake/yamlConfig.cmake"      # [not win]
-    - test -f "${PREFIX}/lib/pkgconfig/yaml-0.1.pc"   # [not win]
```
Comment (jakirkham) [inline recipe/meta.yaml:42]: Should we keep the `pkgconfig` test?

**Review by jakirkham — COMMENTED** [review]
Thanks Tobias! 😄

**Comment by Tobias-Fischer** [comment]
Any more input on this PR (and in particular to my questions above)?

File: recipe/meta.yaml (hunk 1 above)
Comment (Tobias-Fischer) [reply recipe/meta.yaml:42]: Happy either way - let me know what you prefer.

Comment (jakirkham) [reply recipe/meta.yaml:42]: Let's keep the test

**Comment by jakirkham** [comment]
@beckermr @ocefpaf, any thoughts here? 🙂

**Comment by beckermr** [comment]
Let's bump by two and merge once the build passes.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -7,11 +7,9 @@ package:
 source:
   url: https://github.com/yaml/libyaml/archive/{{ version }}.tar.gz
   sha256: fa240dbf262be053f3898006d502d514936c818e422afdcf33921c63bed9bf2e
-  patches:
-    - add-pkg-config.patch
 
 build:
-  number: 1
+  number: 0
```
Comment (jakirkham) [inline recipe/meta.yaml]:
```suggestion
  number: 2
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -37,10 +37,6 @@ test:
     - if not exist "%LIBRARY_LIB%\\yaml.lib" exit 1   # [win]
     - if not exist "%LIBRARY_BIN%\\yaml.dll" exit 1   # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml:39]:
```suggestion

    # Check cmake and pkg-config scripts
    - test -f "${PREFIX}/cmake/yamlConfig.cmake"      # [not win]
    - test -f "${PREFIX}/lib/pkgconfig/yaml-0.1.pc"   # [not win]
```

File: recipe/meta.yaml [hunk 4]
```diff
@@ -40,7 +40,6 @@ test:
     # Check cmake and pkg-config scripts
     - test -f "${PREFIX}/cmake/yamlConfig.cmake"      # [not win]
     - test -f "${PREFIX}/lib/pkgconfig/yaml-0.1.pc"   # [not win]
```
Comment (jakirkham) [inline recipe/meta.yaml:41]:
```suggestion
    - test -f "${PREFIX}/lib/pkgconfig/yaml-0.1.pc"   # [not win]

```

File: recipe/meta.yaml (hunk 3 above)
Comment (Tobias-Fischer) [reply recipe/meta.yaml:39]: This also added the cmake test, which won't pass as it's not a cmake build anymore.

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

**Comment by Tobias-Fischer** [comment]
Fix #22

File: recipe/meta.yaml (hunk 3 above)
Comment (jakirkham) [reply recipe/meta.yaml:39]: Ah ok. Sorry let's drop that then. Thanks for catching this 🙂

**Comment by jakirkham** [comment]
Thanks Tobias! 😄

**Pull request merged**